src/ontology/mondo-edit.obo

```diff
@@ -617825,6 +617846,130 @@ intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.o
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-0587-4693
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8986" xsd:anyURI
 
+[Term]
+id: MONDO:1040071
+name: SLC24A1-related retinopathy
+def: "Any retinopathy caused by variants in the SLC24A1 gene." [https://clinicalgenome.org/working-groups/clinical-domain/ocular-cdwg/, https://orcid.org/0000-0001-5935-2391, https://orcid.org/0000-0002-0587-4693]
+synonym: "SLC24A1-related retinopathy" EXACT CLINGEN_LABEL [https://clinicalgenome.org/working-groups/clinical-domain/ocular-cdwg/, https://orcid.org/0000-0001-5935-2391]
+is_a: MONDO:0019118 {source="https://clinicalgenome.org/working-groups/clinical-domain/ocular-cdwg/", source="https://orcid.org/0000-0001-5935-2391"} ! inherited retinal dystrophy
+intersection_of: MONDO:0019118 ! inherited retinal dystrophy
+intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/10975 ! SLC24A1
+property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-0587-4693
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8986" xsd:anyURI
+
+[Term]
+id: MONDO:1040072
+name: C1QTNF5-related retinopathy
+def: "Any retinopathy caused by a variant in the C1QTNF5 gene." [https://clinicalgenome.org/working-groups/clinical-domain/ocular-cdwg/, https://orcid.org/0000-0001-5935-2391, https://orcid.org/0000-0002-0587-4693]
+synonym: "C1QTNF5-related retinopathy" EXACT CLINGEN_LABEL [https://clinicalgenome.org/working-groups/clinical-domain/ocular-cdwg/, https://orcid.org/0000-0001-5935-2391]
+is_a: MONDO:0019118 {source="https://clinicalgenome.org/working-groups/clinical-domain/ocular-cdwg/", source="https://orcid.org/0000-0001-5935-2391"} ! inherited retinal dystrophy
+intersection_of: MONDO:0019118 ! inherited retinal dystrophy
+intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/14344 ! C1QTNF5
+relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/general/qc-single-child.sparql
```

Comment:
Why does this "excluded QC check annotation" come from? What term is this Mondo term a single child of?